README.md

```diff
@@ -4,19 +4,35 @@
 
 > This a web app built on Rails API as backend and React framework as a frontend. This app is used for reserving a doctor's appointment by patients. Users can create, view, edit and delete a list of appointment(s).
 
-## Demo Video
+## Live Demo Link
 
-[Video Link]()
+[Live Link](https://shielded-wave-72984.herokuapp.com/)
+
+
+## Kanban Board
+
+[Live Link](https://github.com/users/white3d/projects/2/views/1)
+
+
+## API documentation
+
+[Live Link](https://medium.com/@pazzomuk/doctors-appointments-api-documentation-2412e3e9eb3f)
 
-## Live Demo Link
 
-[Live Link]()
+## DB Graph
+- ![](./doctor-appointment-db-graph.png)
+
+doctor-appointment-db-graph
 
 ## Built With
 
-![Ruby](https://icongr.am/devicon/ruby-original.svg?size=50&color=currentColor)
-![Ruby on Rails](https://icongr.am/devicon/rails-original-wordmark.svg?size=50&color=currentColor)
-![PostgreSQL](https://icongr.am/devicon/postgresql-original.svg?size=50&color=currentColor)
+- ![](https://img.shields.io/badge/Github-blueviolet)
+- ![](https://img.shields.io/badge/Ruby-red)
+- ![](https://img.shields.io/badge/Ruby*on*Rails-red)
+- ![](https://img.shields.io/badge/PostgreSql-blue)
+- ![](https://img.shields.io/badge/Webpack-blue)
+- ![](https://img.shields.io/badge/React-blue)
+- ![](https://img.shields.io/badge/Redux-violet)
 
 ## Get Started
```

Comment:
- [ ]  Your instructions about how to run the project are a little confusing because they are not in order.

- Clone project
- Switch to the folder
- `bundle install`
- `rails db:create && rails db:migrate && rails db:seed`
- go to the frontend folder `cd /frontend`
- `yarn install` for frontend dependencies

(In case I missed something bear with me but you get the idea.  Steps in order are more readable for your visitors).

Then what to do?  Run the backend `rails server`? Or the front end `yarn start` or `npm start`? From which folders?

Kindly improve your setup instructions, that way all your visitors (including job recruiters) will be able to smoothly run your project 💪